packages/design-system/src/components/Link/Link.spec.tsx

```diff
@@ -20,6 +19,11 @@ context('<Link />', () => {
 
 	it('should render external', () => {
 		cy.mount(<External />);
+		cy.get('.link .link__icon').should('have.attr', 'name', 'talend-link');
```

Comment:
External links should continue to open in a new tab
```suggestion
		cy.get('.link .link__icon').should('have.attr', 'name', 'talend-link');
		cy.get('.link')
			.should('have.attr', 'title', 'Open in a new tab')
			.should('have.attr', 'target', '_blank');
```

Reply:
But we don't force target="_blank" for external link , so we can have external link that does not open in a new tab

Reply:
Can be painful to get the distinction, I would suggest forcing external links into a new tab

Reply:
I agree lets force it 👍

Reply:
I raised the question to our Product Designers. 
For now, forgot my suggestions.